src/ensembl/src/content/app/browser/track-panel/track-panel-bar/TrackPanelBar.tsx

```diff
@@ -24,13 +25,15 @@ type TrackPanelBarProps = {
 const TrackPanelBar: FunctionComponent<TrackPanelBarProps> = (
   props: TrackPanelBarProps
 ) => {
+  const drawerOpenedForGenome = props.drawerOpened[props.activeGenomeId];
+
   const moveTrackPanel = useCallback(() => {
-    if (props.drawerOpened === true) {
+    if (drawerOpenedForGenome === true) {
```

Comment:
can we, instead of writing `=== true`, write something like `if (isDrawerOpened)`? Firstly, it reads like a sentence; and secondly, typescript won’t allow the value that is declared to be of type boolean to be of any other potentially truthy type. So `if (isDrawerOpened)` will be exactly the same as `if (drawerOpenedForGenome === true)`, but less wordy.

Reply:
I am not really a fan of these truthy and falsy values as it is not used in most static typing languages. However, just to be consistent with the rest of the developers, I'll remove the explicit checks for `true` and `false`.